w01-controlling-the-dom/d4-dawn-intro-jquery/exercises.md

```diff
@@ -0,0 +1,35 @@
+# jQuery and DOM Exercises
+
+### Docs & Resources
+
+* [Document Object Model docs (Mozilla)](https://developer.mozilla.org/en-US/docs/Web/API/document)
+* [Document Object Model docs (W3Schools)](http://www.w3schools.com/jsref/dom_obj_document.asp)
+* [List of DOM Events](https://developer.mozilla.org/en-US/docs/Web/Events)
+
+### Exercise 0: Traversing the DOM
+
+1. Go to https://generalassemb.ly and open your Dev Tools. Select the `html` element and look at its children. What type of data structure of this collection of elements?
```

Comment:
wording is confusing: "What type of data structure of this collection of elements?"